src/entity/Comment.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Projet5\entity;
+
+/**
+ * Comment
+ */
+class Comment
+{
+    private $id;
+    private $contents;
+    private $date_comment;
+    private $publish;
+    private $id_bpf_blog_posts;
+    private $id_bpf_users;
+
+    /**
+     * Method getId
+     * 
+     * Get value of idComment
+     *
+     * @return void
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Method setId
+     * 
+     * Set value of idComment
+     *
+     * @param int $id
+     *
+     * @return void
+     */
+    public function setId($id): void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * Method getContents
+     * 
+     * Get value of contents
+     *
+     * @return void
+     */
+    public function getContents()
+    {
+        return $this->contents;
+    }
+
+    /**
+     * Method setContents
+     * 
+     * Set value of Contents
+     *
+     * @param string $contents
+     *
+     * @return void
+     */
+    public function setContents($contents)
+    {
+        $this->contents = $contents;
+    }
+
+    /**
+     * Method getDateComment
+     * 
+     * Get value of DateComment
+     *
+     * @return void
+     */
+    public function getDateComment()
+    {
+        return $this->date_comment;
```

Comment:
idem pour tout les setters de ta classe

Reply:
return $this;    

Ajouté à tous les setters